itou/communications/cache.py

```diff
@@ -0,0 +1,29 @@
+from datetime import date, datetime
+
+from django.core.cache import cache
+
+from itou.communications.models import AnnouncementCampaign
+
+
+CACHE_ACTIVE_ANNOUNCEMENTS_KEY = "active-announcement-campaign"
+
+
+def update_active_announcement_cache():
+    campaign_qs = AnnouncementCampaign.objects.filter(
+        start_date=date.today().replace(day=1), live=True
+    ).prefetch_related("items")
```

Comment:
A mon avis, il serait intéressant de rajouter un `Exists` sur les `AnnouncementItem` histoire de ne pas afficher une campagne vide.

Reply:
Dans l'utilisation du valeur cache on verifie ça : `"active_campaign_announce": (campaign if campaign is not None and campaign.items.count() else None)`

C'est peut-être invalide d'avoir une campagne `live` sans articles et dans ce cas je préférerais ajouter une contrainte qui empêche que ces campagnes peuvent être mis en `live`